pkgs/applications/networking/pyload-ng/default.nix

```
@@ -0,0 +1,56 @@
+{ lib, python3 }:
+
+python3.pkgs.buildPythonApplication rec {
+  version = "0.5.0b3.dev69";
+  pname = "pyload-ng";
+  format = "pyproject";
+
+  disabled = python3.pkgs.pythonOlder "3.8";
+
+  src = python3.pkgs.fetchPypi {
```

```suggestion
  src = python3.fetchPypi {
```
that was recently deprecated

---

python3.fetchPypi doesn't seem to exist

---

probably pkgs.fetchPypi https://github.com/NixOS/nixpkgs/pull/234043

```suggestion
  src = fetchPypi {
```